source/Visual.h

```diff
@@ -29,7 +29,8 @@ class Effect;
 class Visual : public Body {
 public:
 	Visual() = default;
-	Visual(const Effect &effect, Point pos, Point vel, Angle facing, Point hitVelocity = Point());
+	Visual(const Effect &effect, Point pos, Point vel, Angle facing, Point hitVelocity = Point(),
+		double zoom = 1.);
```

Comment:
I count 116 characters.
```suggestion
	Visual(const Effect &effect, Point pos, Point vel, Angle facing, Point hitVelocity = Point(), double zoom = 1.);
```